scripts/quickstart.sh

```diff
@@ -361,9 +373,9 @@ function setup {
 
   # Pull the docker images.
   echo -e "\nPull the docker images..."
-  DOCKER_IMAGE_PARITY="$DOCKER_IMAGE_PARITY:$DOCKER_IMAGE_FUSE_PARITY_VERSION"
-  DOCKER_IMAGE_NETSTAT="$DOCKER_IMAGE_NETSTAT:$DOCKER_IMAGE_NET_STATS_VERSION"
-  DOCKER_IMAGE_APP="$DOCKER_IMAGE_APP:$DOCKER_IMAGE_FUSE_APP_VERSION"
+  DOCKER_IMAGE_PARITY="$DOCKER_IMAGE_PARITY$TEST_NET_STRING:$DOCKER_IMAGE_FUSE_PARITY_VERSION"
+  DOCKER_IMAGE_NETSTAT="$DOCKER_IMAGE_NETSTAT$TEST_NET_STRING:$DOCKER_IMAGE_NET_STATS_VERSION"
+  DOCKER_IMAGE_APP="$DOCKER_IMAGE_APP$TEST_NET_STRING:$DOCKER_IMAGE_FUSE_APP_VERSION"
```

Comment:
If you rename this to `TEST_NET_SUFFIX` I think it'll be more clear why we add this here